preprocessing/nextclade/tests/test.py

```diff
@@ -304,23 +353,30 @@ def sort_annotations(annotations: list[ProcessingAnnotation]):
 
 
 accepted_authors = {
-    "Xi, L.; Yu, X.;": "Xi, L.; Yu, X.;",
-    "Xi,L;Yu,X.;": "Xi, L.; Yu, X.;",
-    "Xi,;Yu,X.;": "Xi, ; Yu, X.;",
-    "Xi, ;Yu,X.;": "Xi, ; Yu, X.;",
-    "Xi,;": "Xi, ;",
-    "Smith, Anna Maria; Perez, Jose X.;": "Smith, Anna Maria; Perez, Jose X.;",
-    "Smith,Anna Maria;Perez,Jose X;": "Smith, Anna Maria; Perez, Jose X.;",
+    "Xi, L.; Yu, X.;": "Xi, L.; Yu, X.",
+    "Xi,L;Yu,X.;": "Xi, L.; Yu, X.",
+    "Xi,;Yu,X.;": "Xi, ; Yu, X.",
+    "Xi, ;Yu,X.;": "Xi, ; Yu, X.",
+    "Xi, ;Yu,X.": "Xi, ; Yu, X.",
+    "Xi,;": "Xi,",
+    "Xi,": "Xi,",
+    "Smith, Anna Maria; Perez, Jose X.;": "Smith, Anna Maria; Perez, Jose X.",
+    "Smith,Anna Maria;Perez,Jose X;": "Smith, Anna Maria; Perez, Jose X.",
+    "VAN DER MEER, F.; DU PONT, P. J.": "van der Meer, F.; du Pont, P. J.",
+    "DE LA CRUZ, A.; LE BLANC, B.; VON TRAPP, U.;": "de la Cruz, A.; le Blanc, B.; von Trapp, U.",
+    "de souza, a.": "de souza, a.",
```

Comment:
Initials should still be capitalized
```suggestion
    "de souza, a.": "de souza, A.",
```